templates/account.php

```diff
@@ -590,14 +590,100 @@ class="fs-tag fs-<?php echo $fs->can_use_premium_code() ? 'success' : 'warn' ?>"
 										</tr>
 										</thead>
 										<tbody>
-										<?php $odd = true;
+										<?php
+                                            $odd = true;
+
+                                            $installed_addons_ids_map = array_flip( $installed_addons_ids );
+
+                                            $fs_options = FS_Options::instance( WP_FS__ACCOUNTS_OPTION_NAME );
+
+                                            $sites = $fs_options->get_option( 'sites', array() );
+
+                                            $plugins_data = null;
+                                            $all_plans    = null;
+                                            $licenses     = null;
+
 											foreach ( $addons_to_show as $addon_id ) {
+                                                $addon              = $fs->get_addon( $addon_id );
+                                                $slug               = $addon->slug;
+											    $is_addon_connected = false;
+											    $addon_info         = array();
+
+                                                if ( isset( $sites[ $slug ] ) ) {
+                                                    $site   = $sites[ $slug ];
+                                                    $plugin = FS_Plugin_Manager::instance( $addon_id )->get();

```

Comment:
@fajardoleo are you sure that `FS_Plugin_Manager::instance( $addon_id )` will always return the `FS_Plugin` instance when this line is executed? If not, we need to address the case when the result is `false`, if yes, please explain why.